api/src/routers/meals.js

```diff
@@ -0,0 +1,82 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    const [allMeals] = await knex.raw("SELECT * FROM Meal");
+    res.json(allMeals);
+  } catch (err) {
+    res.status(500).json({ error: err.message });
+  }
+});
+
+router.post("/api/search", async (req, res) => {
+  if (!req.body.title || req.description.length < 3) {
+    return res
+      .status(400)
+      .send("Missing required fields: name, price, description");
+  }
+  try {
+    const newMeal = req.body;
+    const [id] = await knex("Meal").insert(newMeal);
+    res.status(201).json({ id, ...newMeal });
+  } catch (error) {
+    res.status(500).json({ error: error.message });
+  }
+  const newMeal = {
```

Comment:
why is this hardcoded here? I would imagine the response should send back the answer from db (if not just a simple success message :))